app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for data in people:
+        person = Person(data["name"], data["age"])
+        if "wife" in data and data["wife"] is not None:
+            if data["wife"] in person.__class__.people:
+                person.wife = person.__class__.people[data["wife"]]
+                person.wife.husband = person
+        elif "husband" in data and data["husband"] is not None:
+            if data["husband"] in person.__class__.people:
+                person.husband = person.__class__.people[data["husband"]]
+                person.husband.wife = person
```

Comment:
u have pretty similar parts here, could u optimize this ?
hint: u can iterate through the ['husband', 'wife'] list and use str as keys